src/libs/runtime-utils.js

```diff
@@ -214,17 +214,25 @@ export const isCurrentGalaxyDiskDetaching = apps => {
   return currentGalaxyApp && _.includes(currentGalaxyApp.status, ['DELETING', 'PREDELETING'])
 }
 
-export const collapsedRuntimeStatus = runtime => {
+export const getGalaxyCostTextChildren = (app, galaxyDataDisks) => {
+  const dataDisk = currentAttachedDataDisk(app, galaxyDataDisks)
+  return app ?
+    [getConvertedAppStatus(app.status), dataDisk?.size ? ` (${Utils.formatUSD(getGalaxyCost(app, dataDisk.size))} / hr)` : ``] : ['None']
+}
+
+export const getConvertedRuntimeStatus = runtime => {
   return runtime && (runtime.patchInProgress ? 'LeoReconfiguring' : runtime.status) // NOTE: preserves null vs undefined
 }
 
 export const isAppDeletable = app => _.includes(app?.status, ['RUNNING', 'ERROR'])
 
-export const convertedAppStatus = appStatus => {
+export const getConvertedAppStatus = appStatus => {
   return Utils.switchCase(_.upperCase(appStatus),
-    ['STOPPED', () => _.capitalize('PAUSED')],
-    ['STOPPING', () => _.capitalize('PAUSING')],
-    ['STARTING', () => _.capitalize('RESUMING')],
+    ['STOPPED', () => 'Paused'],
+    ['STOPPING', () => 'Pausing'],
+    ['STARTING', () => 'Resuming'],
+    ['PRESTARTING', () => 'Starting'],
+    ['PRESTOPPING', () => 'Stopping'],
```

Comment:
```suggestion
    ['PRESTARTING', () => 'Preresuming'],
    ['PRESTOPPING', () => 'Prepausing'],
```

Reply:
Ah. I changed these to 'Resuming' and 'Pausing'. The pre- statuses aren't intended for display to users.

Reply:
I still see 

```
['PRESTARTING', () => 'Starting'],
['PRESTOPPING', () => 'Stopping'],
```